.kitchen.yml

```diff
@@ -1,78 +1,37 @@
 ---
 driver:
-  name: dokken
+  name: vagrant
```

Comment:
I reverted back to `dokken` per @chris-rock and @bigbam505 suggestions. There is an alternative for vagrant called `.kitchen.vagrant.yml`